### ISSUE TITLE
Accessibility review and remediation

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/pre-assembly#1469]
- Run SiteImprove, [axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd), and/or [WAVE](https://wave.webaim.org/). Correct any identified issues, focusing on WCAG A and AA violations in order of severity.
- Review the Developer Checklists [from DLSS](https://docs.google.com/document/d/14vsLwKB4BiZZnkcsrj8r9hAa9w1hHS6IvD6sKDr7jPo/edit?usp=sharing#heading=h.oqlgf78ham6l) and/or [Deque University](https://dequeuniversity.com/class/program-management/jobs-roles/developers/checklist-developers). (Deque University is available to Stanford staff by [requesting an account](https://uit.stanford.edu/accessibility/help/training-modules).)

**Comment by dnoneill** [comment]
Do we need to do more on this ticket? All axe tests are now passing after updating the application to comply. WAVE has two redundant links warnings and no errors. I have a PR for one small problem I found.

**Comment by jmartin-sul** [comment]
We think this is close to done for Preassembly.  A11y tests were added to the automated test suite and are passing.  Manual review of keyboard accessibility by Alan looks good.  SiteImprove plugin did report two level A compliance issues, which Alan will add to this ticket.

Argo is a whole other beast, in part because it's built on Blacklight, so at least some issues would have to be addressed upstream (ideally, or hacked around locally).  That's a separate ticket outside the scope of this work cycle.

**Comment by alundgard** [comment]
The SiteImprove browser extension reports two occurrences of a Level A compliance issue. These only appear when the Content language selector is visible on the page.
Steps to reproduce:
- Select the relevant form options so that the Content language selector appears.
- Run the SiteImprove browser extension. SiteImprove shows the following info.
> Hidden element has focusable content. A page element with the attribute `aria-hidden="true"` contains focusable elements (elements that can receive focus from a keyboard).

I believe the solution is to remove `aria-hidden="true"` on the magnifying glass and carat buttons.

**Comment by dnoneill** [comment]
That is correct. This change was added to the https://github.com/sul-dlss/pre-assembly/pull/1485

**Comment by peetucket** [comment]
I think we can close this ticket now?
